modules/yieldmoBidAdapter.js

```diff
@@ -93,6 +93,9 @@ export const spec = {
         if (request.schain) {
           serverRequest.schain = JSON.stringify(request.schain);
         }
+        if (request.params && request.params.lr_env) {
```

Comment:
@ym-atsymuk minor: I suggest using `utils.deepAccess(request, 'params.lr_env')` for such cases